rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -314,6 +314,209 @@ pub async fn add_wasms_to_sns_wasm(
     })
 }
 
+/// A builder for fine-tuning and installing the NNS canister suite in PocketIc.
+pub struct NnsInstaller {
```

Comment:
This supersedes install_nns_canisters, right? Maybe, add a comment that install_nns_canisters is deprecated?

Reply:
Will be done in https://github.com/dfinity/ic/pull/3696